docs/package_info/praw7_migration.rst

```diff
@@ -0,0 +1,62 @@
+Migrating to PRAW 7.X
+=====================
+
+Exception Handling
+------------------
+
+Class :class:`.APIException` has also been renamed to
+:class:`.RedditAPIException`. Importing :class:`.APIException` will still work,
+but is deprecated, but will be removed in PRAW 8.0.
+
+PRAW 7 introduced a fundamental change in how exceptions are recieved from
+Reddit's API. Reddit can return multiple exceptions for one API action, and
+as such, the exception :class:`.RedditAPIException` serves as a container for
+each of the true exception objects. These objects are instances of
+:class:`.RedditErrorItem`, and they contain the information of one "error" from
+Reddit's API. They have the three data attributes that :class:`.APIException`
+used to contain.
+
+Most code regarding exceptions can be quickly fixed to work under the new
+system. All of the exceptions are stored in the ``items`` attribute of the
+exception as entries in a list. In the example code below, observe how
+attributes are accessed.
+
+.. code-block:: python
+
+    try:
+        reddit.subreddit("test").submit("Test Title", url="invalidurl")
+    except RedditAPIException as exception:
```

Comment:
Should this show how it was done previously via `APIException`?

Reply:
Probably should.